JiraPS/Public/Get-JiraVersion.ps1

```diff
@@ -11,60 +12,66 @@
        Get-JiraVersion -ID '66596'
        This example returns information about all JIRA Version visible to the current user (or using anonymous access if a PSJira session has not been defined) for the project.
     .INPUTS
-       This function does not accept pipeline input pending class creation.
+        [PSJira.Project]
     .OUTPUTS
        This function outputs a PSobject(s).
     .NOTES
        This function requires either the -Credential parameter to be passed or a persistent JIRA session. See New-JiraSession for more details.  If neither are supplied, this function will run with anonymous access to JIRA.
     #>
-    [CmdletBinding(DefaultParameterSetName = 'Project')]
+    [CmdletBinding(DefaultParameterSetName = 'Key')]
```

Comment:
I would like parametersets to be called `by*`.
I believe it makes for easier reading in the help.